azure-pipelines.yml

```diff
@@ -42,15 +42,15 @@ steps:
         sonar.cfamily.build-wrapper-output.bypass=true
     condition: and(succeeded(), eq(variables['System.PullRequest.IsFork'], 'False'))
 
-  - script: npm i
-    displayName: 'npm install'
-
-  - script: npm ci --prefix storybook/storybook
+  - script: cd storybook/storybook && npm ci
     displayName: 'npm install storybook'
 
-  - script: npm ci --prefix storybook/design-system
+  - script: cd storybook/design-system && npm ci
```

Comment:
why do you change this ? this did'nt work ?

Reply:
I think it's was the cause of bad dependencies installation. But not.
But it's more simple command and this use case of prefix seems to not do what we want

Reply:
ok